src/types/Venue.ts

```diff
@@ -21,7 +21,7 @@ export enum RoomVisibility {
   nameCount = "count/name",
 }
 
-export type AnyRoom = RoomData | CampRoomData;
+export type AnyRoom = RoomData | CampRoomData | AvatarGridRoom;
```

Comment:
just an FYI `AvatarGridRoom` is part of `avatargrid` template which will go soon.

Reply:
_nods_ all good. This is allowing the types to work for the changes made here in the meantime. Once we 💥  Avatar grid, this can get removed at the same time.